content/install-hook-userstylesworld.js

```diff
@@ -12,17 +12,35 @@
 
   const onPageLoaded = event => {
     if (event.data
-    && event.data.type === 'usw-ready'
     && allowedOrigin === event.origin
     ) {
-      sendPostMessage({type: 'usw-remove-stylus-button'});
+      if (event.data.type === 'usw-ready') {
+        sendPostMessage({type: 'usw-remove-stylus-button'});
 
-      if (location.pathname === '/api/oauth/style/new') {
-        const styleId = Number(new URLSearchParams(location.search).get('vendor_data'));
-        API.styles.get(styleId).then(style => {
-          style.sourceCode = style.tmpSourceCode;
-          sendPostMessage({type: 'usw-fill-new-style', data: style});
-        });
+        if (location.pathname === '/api/oauth/style/new') {
+          const styleId = Number(new URLSearchParams(location.search).get('vendor_data'));
+          API.styles.get(styleId).then(style => {
```

Comment:
This part is completely different in the upcoming #1285, which we'll merge first.

Reply:
No problems, I can rebase on the main branch once it's merged.

Reply:
It's merged.

Reply:
Rebased the code, should be good now.